frontend/src/components/table/TableRow.js

```diff
@@ -560,7 +560,7 @@ class TableRow extends PureComponent {
                 colIndex={idx}
                 hasComments={!!(hasComments && idx === 0)}
                 cellExtended={cellsExtended}
-                key={`${rowId}-${property}`}
+                key={`${rowId}-${property}-${uuidv4()}`}
```

Comment:
@petrican  Q: why not using the `${idx}` instead of `${uuidv4()}`?

Reply:
@teosarca using indexes from map is an anti-pattern. 

<img width="1144" alt="Screenshot 2022-01-19 at 09 02 52" src="https://user-images.githubusercontent.com/1708561/150080669-8a270f6c-f245-40c6-8e63-c96ce824bf55.png">

https://medium.com/wesionary-team/using-index-as-a-key-is-an-anti-pattern-in-react-8e5db3aea3a9

Reply:
@petrican  I wasn't suggesting to use just the `idx`. I was suggesting to use `${rowId}-${property}-${idx}`.

Ahh, and now that i am looking deeper, i am even not sure why the `${rowId}` needs to be part of the key because as far as I understand the `key` needs to be unique inside the list which contains it.

In our case, the list consists of the cells which are part of a given TableRow.
So, ofc, all cells from that row will share the same `rowId`.
So it's pointless to have it there as a discriminator because it's the same.

To conclude, I would go with `${property}-${idx}`.

Reply:
Yeah. That works too. Can the PR be merged ?